packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -302,6 +325,18 @@ class OpPerfTelemetry {
 				opPerfData: {},
 			});
 		}
+
+		if (message.type == MessageType.NoOp) {
+			// Count the number of no-ops submitted by this client.
+			// The value is reset when we log the OpStats sampled event.
+			this.noOpCount++;
+		}
+
+		this.opsLogger.sendPerformanceEvent({
```

Comment:
I think you should add these two proerties under a `details` object for the following reasons:
1. You can see all interesting properties in one column in Kusto. You also don't need to worry about creating additional columns (which has problems like the one mentioned in the next point).
2. We have run out of columns in Kusto currently, so this will go in `Payload` property. Might as well add it to an existing and known property.

This would look like:
this.opsLogger.sendPerformanceEvent({
   details: {
      submitOpCount: OpPerfTelemetry.OPS_SAMPLE_RATE,
	  noOpCount: this.noOpCount,
   }
});

Our telemetry loggers will convert the details object into a json string before logging.